tools/add_triage_signature.py

```diff
@@ -878,7 +878,11 @@ class MediaDisconnectionSignature(ErrorSignature):
     Eventually, the signature will include common i/o errors and the number of times they appeared
     '''
 
-    ERRORS_PATTERNS = 'Unable to read from the discovery media'
+    ERRORS_PATTERNS = [
+        'media was likely disconnected',
+        'Unable to read from the discovery media',
+        'Error: decoding and writing image',
```

Comment:
I remotely remember something about it not being media disconnection but rather something else
@omertuc can you take a look?